src/tests/ftest/unittest/unittest.py

```diff
@@ -69,7 +69,7 @@ def test_smd_ut(self):
                   nvme_get_pool, nvme_set_pool_info, nvme_add_pool,
                   nvme_get_device, nvme_set_device_status,
                   nvme_add_stream_bond, nvme_get_stream_bond
-        :avocado: tags=all,unittest,pr,tiny,hw,smd_ut
+        :avocado: tags=all,unittest,tiny,regression,vm,smd_ut
```

Comment:
How should we handle the pending change in #574, where the tags should eventually be `all,unittest,tiny,pr,smd_ut`?  

Should this PR be delayed and updated after it lands or should this test be added to #574?

Reply:
IMHO, this one is ready to land.  #574 is a WIP with an unknown ETA to landing.  This patch helps a pain point we a suffering right now.

#574 has been ongoing for long enough that it's going to have to merge master (and resolve conflicts) already anyway.

Reply:
Ok